spec.md

```diff
@@ -654,7 +654,10 @@ An asynchronous response triggers the Platform to poll the endpoint
 indicates that the requested operation has succeeded or failed. Service Brokers
 MAY include a status message with each response for the `last_operation`
 endpoint that provides visibility to end users as to the progress of the
-operation.
+operation. For Service Brokers which can't return the resource message in
+asynchronous response, they SHOULD guarantee resource message included in the
+response of the `last_operation` endpoint when the asynchronous operation
+completed.
```

Comment:
I think we'd need to make it clear that even if the dashboard_url isn't ready yet, they still need to send something because the spec requires it - even if the URL is temporary.

Reply:
So basically you mean the platform would check the dashboard_url even if the state is` in progress`? Or service broker can return an empty one if they are not ready?

Reply:
@duglin Got it, what I meant is service brokers can pass anything they want before the operation is completed, but they have to make the right one included if the `dashborad_url` exists. Does that ok to you? I will update the spec, thanks!